engine/src/main/resources/org/terasology/engine/logback/filter-reflections.xml

```diff
@@ -0,0 +1,14 @@
+<included>
+    <!-- Expects to be included within an <appender>
+         (or something else that can contain filters). -->
+
+    <!-- Requires org.terasology.logback.RegexFilterAction; see setup.xml -->
+
+    <!-- WONTFIX: gestalt routinely builds empty configs and then adds to them later. -->
```

Comment:
I like the `WONTFIX` comment - helps to understand that something is known but intentionally left as is :+1: